QualityControl/public/common/filters/filterViews.js

```diff
@@ -51,16 +51,22 @@ const createFilterElement = (config, filterMap, onInputCallback, onEnterCallback
  * @returns {vnode} - virtual node element
  */
 export function filtersPanel(filterModel, pageModel) {
-  const { filterMap, setFilterValue, filterService } = filterModel;
+  const { filterMap, setFilterValue, filterService, visible, clearFilter } = filterModel;
   const onInputCallback = setFilterValue.bind(filterModel);
   const onChangeCallback = setFilterValue.bind(filterModel);
   const onEnterCallback = () => filterModel.triggerFilter(pageModel);
+  const clearFilterCallback = clearFilter.bind(filterModel);
   const filtersList = filtersConfig(filterService);
 
+  if (!visible) {
+    return null;
+  }
+
   return h(
-    '.w-100.flex-row.p2.g2',
+    '.w-100.flex-row.p2.g2.justify-center#filterElement',
     [
       triggerFiltersButton(onEnterCallback),
+      clearFiltersButton(clearFilterCallback),
```

Comment:
Clicking clear, does not remove value from input of RunNumber or from dropdown unless another action is performed.
e.g. 
- set RunNumber to 42
- set RunType to something 
- click Clear --- nothing happens
- select a different RunType and at this point RunNumber is reset